### ISSUE TITLE
RETURN clause throws OCommandExecutorNotFoundException with SQL batch script

### ISSUE THREAD

**Issue by benjaminbillet** [orientechnologies/orientdb#6712]
## OrientDB Version, operating system, or hardware.
v2.2.10 MacOSX Java 

## Expected behavior and actual behavior
According to http://orientdb.com/docs/master/SQL-batch.html the RETURN clause can be used to return the result of a query. However, it fails with an OCommandExecutorNotFoundException

## Steps to reproduce the problem
Consider the following Java snippet:

```java
OrientGraph graph = ...;

String script = "return 'hello'";
Object result = graph.command(new OCommandScript("sql", script)).execute();
System.out.println(result);
// prints "hello" (fine)

script = "let a = select from V\n"
		+ "return $a";
result = graph.command(new OCommandScript("sql", script)).execute();
System.out.println(result);
// prints com.tinkerpop.blueprints.impls.orient.OrientDynaElementIterable@55a147cc (fine)

script = "return (select from V)";
result = graph.command(new OCommandScript("sql", script)).execute();
System.out.println(result);
// fails, with the following exception: 

Exception in thread "main" com.orientechnologies.orient.core.command.OCommandExecutorNotFoundException: Cannot find a command executor for the command request: sql.(SELECT FROM V)
	DB name="testDatabase"
	DB name="testDatabase"
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.throwSerializedException(OChannelBinaryAsynchClient.java:427)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.handleStatus(OChannelBinaryAsynchClient.java:378)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.beginResponse(OChannelBinaryAsynchClient.java:260)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.beginResponse(OChannelBinaryAsynchClient.java:152)
	at com.orientechnologies.orient.client.remote.OStorageRemote.beginResponse(OStorageRemote.java:2087)
	at com.orientechnologies.orient.client.remote.OStorageRemote$27.execute(OStorageRemote.java:1137)
	at com.orientechnologies.orient.client.remote.OStorageRemote$2.execute(OStorageRemote.java:192)
	at com.orientechnologies.orient.client.remote.OStorageRemote.baseNetworkOperation(OStorageRemote.java:224)
	at com.orientechnologies.orient.client.remote.OStorageRemote.networkOperation(OStorageRemote.java:189)
	at com.orientechnologies.orient.client.remote.OStorageRemote.command(OStorageRemote.java:1113)
	at com.orientechnologies.orient.core.command.OCommandRequestTextAbstract.execute(OCommandRequestTextAbstract.java:69)
	at com.tinkerpop.blueprints.impls.orient.OrientGraphCommand.execute(OrientGraphCommand.java:49)
	at Test2.main(Test2.java:38)
```

Note that "return (select $a as something)" fails as well.

**Comment by luigidellaquila** [comment]
Hi @benjaminbillet 

It seems to be a bug, I'll take a look at it asap

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @benjaminbillet 

I just pushed a fix for this in 2.2.x branch. The fix will be released with 2.2.11

I'm closing this, please reopen if you see any related problems

Thanks

Luigi
